services/repository-managers/src/test/java/org/sagebionetworks/repo/manager/authentication/PersonalAccessTokenManagerImplUnitTest.java

```diff
@@ -131,13 +146,47 @@ void testIssueToken() {
 		verify(mockPersonalAccessTokenDao).deleteLeastRecentlyUsedTokensOverLimit(userInfo.getId().toString(), EXPECTED_TOKEN_LIMIT);
 	}
 
+	@Test
+	void testIssueTokenLimitedAccessTokenScope() {
```

Comment:
You might want to add the test to check on the removeAll for double authorize in the list edge case